app1/webpack.config.js

```diff
@@ -53,10 +53,10 @@ const webpackConfig = {
           eager: true,
         },
         "@chakra-ui/react": {
-          singleton: true,
-          shareKey: "chakra",
-          shareScope: "modern",
-          requiredVersion: deps["@chakra-ui/react"],
+          singleton: false,
+          version: deps["@chakra-ui/react"],
+          // requiredVersion: deps["@chakra-ui/react"],
+          strictVersion: false,
```

Comment:
interesting! 

why do I need to use shareKey and shareScope? the react versions examples use it

Reply:
I am not sure but you might need them when you have multiple remotes and you want to separate the sharing by scope. I personally never used it and never seen a usecase.